### PR TITLE
[exporter][pulsar][doc] Add pulsarexporter README.md

### PR DESCRIPTION
**Pull request by tjiuming**
**Description:** <Describe what has changed.>
add README.md for pulsarexporter

**Link to tracking Issue:** <Issue number if applicable>
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/9795

**Testing:** <Describe what testing was performed and which tests were added.>

NO NEED

### CODE REVIEW

**Comment by tjiuming** [comment]
@dmitryax  Please help review

File: exporter/pulsarexporter/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Pulsar Exporter
+
```
Comment (dmitryax) [inline exporter/pulsarexporter/README.md:2]: Please add a standard doc header with "in development" status. Check how it's done for other components

Comment (tjiuming) [reply exporter/pulsarexporter/README.md:2]: RESOLVED, please help approve test again. @dmitryax

File: exporter/pulsarexporter/README.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+| Status                   |                       |
+| ------------------------ |-----------------------|
+| Stability                | [in-development]      |
```
Comment (codeboten) [inline exporter/pulsarexporter/README.md:5]: Please add the link at the bottom, see https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/c85ffc0da0463c37975fbad94dd3a002363d17a6/exporter/parquetexporter/README.md?plain=1#L32 as an example

Comment (tjiuming) [reply exporter/pulsarexporter/README.md:5]: RESOLVED

**Review by dmitryax — APPROVED** [review]

**Comment by tjiuming** [comment]
all the CI checks passed, please help merge the PR

**Pull request merged**